assets/css/rainbow.css

```diff
@@ -6,11 +6,15 @@
     display: inline-block;
     float: left;
     text-align: left !important;
-    margin-top: 6px;
+    margin-top: 1px;
```

Comment:
Is there any reason to set this so low. I think it looks better if there's more space between the logo and the top of the page.